### ISSUE TITLE
Adding "hash" links to markdown headers

### ISSUE THREAD

**Issue by isaacrg** [sintaxi/harp#328]
Would it be possible to make the markdown header tags (h1 -> h6) to have hash links automatically added?

**I.E.**
```md
# Test Link
```

Should go to:

```html
<a  name="test-link" id="test-link"></a><h1>Test Link</h1>
```

This would be incredibly helpful, in my opinion. Thoughts?

**Comment by mrchief** [comment]
This would be great! Is this in v1.0.0 release?

**Comment by mrchief** [comment]
@sintaxi @kennethormandy Is this fix available in 1.0.0 release or is this in o.9.2? I don't see it happening in 0.9.2 so I'm assuming v1.0.0 will have this?

Also, can we have v1.0.0 out soon please??

**Comment by cxreg** [comment]
I'm not sure why this issue is closed.  It looks as if the `id` rendering was "temporarily" disabled several years ago with the intent to re-enable it later?  It's been quite a while, should this issue be re-opened so it can be added?
